constraints-dev.txt

```diff
@@ -1,4 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
+click<=8.1.8
```

Comment:
please add a comment above linking to the issue so that we know when we can move it forward.